### PR TITLE
Run test on pr 

### PR DESCRIPTION
**Pull request by AtmegaBuzz**

### CODE REVIEW

File: website/views.py [hunk 1]
```diff
@@ -558,7 +558,10 @@ def get_initial(self):
     def post(self, request, *args, **kwargs):
         
         # resolve domain
-        url = request.POST.get("url").replace("www.","https://")
+        url = request.POST.get("url")
```
Comment (fredfalcon) [inline website/views.py:561]: We need to remove www and http for all domains so they are stored the same in the db.

**Review by fredfalcon — CHANGES_REQUESTED** [review]
Please change this back to how it was and only add the exception for bugheist

File: website/views.py (hunk 1 above)
Comment (AtmegaBuzz) [reply website/views.py]: resolved

**Review by fredfalcon — APPROVED** [review]

**Comment by fredfalcon** [comment]
Thank you!

**Pull request merged**